app/util/autoversioning.py

```diff
@@ -20,18 +20,24 @@ def get_version():
     :return: The version of the application
     :rtype: str
     """
-    if getattr(sys, 'frozen', False):
-        return _get_frozen_package_version()  # frozen/packaged
 
-    return _calculate_source_version()  # unfrozen/running from source
+    return _get_frozen_package_version() or _calculate_source_version() or '0.0.0'
 
 
 def _try_rename(src, dst):
```

Comment:
I would like to keep the rename logic for now to keep the backup and see the version change/difference.
But in future, I agree that we can clean this and make `write_package_version_file` to remove old file.